fixtures/test.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-env jest */
+import request from 'supertest';
+import server from '../server/server';
+import config from '../yeep.config';
+import createSession from '../server/api/session/create/service';
+
+describe('Test login', () => {
```

Comment:
Shouldn't this read "Fixtures" instead of "Test login"?

Reply:
This file was for internal use and i would be destroying it pre-PR